docs/trouble-shooting/network.md

```diff
@@ -0,0 +1,205 @@
+# Some tips for digging network issues between Worker and PeerPod
+
```

Comment:
@huoqifeng thanks a lot for sending this PR. Very helpful.
Just minor nits.

-  trouble-shooting => troubleshooting
-  Since some of the instructions are specific to calico and will differ for other CNIs, I think it might make sense to rename this file as network-calico.md ?

Reply:
Thank you @bpradipt it's changed according to the comments.